docker/ubuntu_18.04-dev.dockerfile

```diff
@@ -162,8 +166,8 @@ RUN if [ "$build_pyne" = "YES" ]; then \
     fi
 ENV PATH $HOME/.local/bin:$PATH
 RUN if [ "$build_pyne" = "YES" ]; then \
-        cd $HOME \
-        && nuc_data_make ; \
+        cd $HOME/opt/pyne \
```

Comment:
We are intentionally using the version of `nuc_data_make` that is in the path and has been installed, rather than the version that is in the repository.  This is meant to double check that the process is carried out consistently with the version that has been installed.

Reply:
My mistake @gonuke. I didn't consider it `ENV PATH $HOME/.local/bin:$PATH`. Should I revert the change @gonuke?